satellite/satellitedb/overlaycache.go

```diff
@@ -329,7 +329,14 @@ func (cache *overlaycache) CreateStats(ctx context.Context, nodeID storj.NodeID,
 		}
 	}
 
-	return getNodeStats(nodeID, dbNode), Error.Wrap(tx.Commit())
+	// TODO: Allegedly tx.Get_Node_By_Id and tx.Update_Node_By_Id should never return a nil value for dbNode,
```

Comment:
The implementation of [`tx.Update_Node_By_Id` returns nil when `QueryRow` returns `sql.ErrNoRows`](https://github.com/storj/storj/blob/master/satellite/satellitedb/dbx/satellitedb.dbx.go#L5754) so it looks an expected behavior for DBX update generated queries.